Gemfile

```diff
@@ -38,7 +38,7 @@ gem 'local_time'
 gem 'pagy'
 
 # deprecated
-gem 'record_tag_helper'
+gem 'record_tag_helper', github: 'mmoll/record_tag_helper', branch: 'update_gemspec'
```

Comment:
Maybe you want to copy the three methods this gem supplies to a helper file?

Reply:
I like this idea a lot, thanks.